Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_smorci/src/main.rs

```diff
@@ -0,0 +1,72 @@
+use num_bigint::{BigUint, ParseBigIntError};
+use std::io;
+use std::str::FromStr;
+
+const NUMBER_OF_DAYS_IN_A_YEAR: u32 = 365;
+
+fn convert_years_to_days(years: BigUint) -> BigUint {
+    NUMBER_OF_DAYS_IN_A_YEAR * years
+}
+
+fn convert_string_to_biguint(string: &str) -> Result<BigUint, ParseBigIntError> {
+    let result = BigUint::from_str(string.trim())?;
+
+    Ok(result)
+}
+
+fn get_input() -> io::Result<String> {
+    let mut buffer = String::new();
+
+    let stdin = io::stdin();
+
+    stdin.read_line(&mut buffer)?;
+
+    Ok(buffer)
+}
+
+fn main() {
+    println!("Welcome to age converter! This program converts ages to days.");
+    println!();
+    println!("Enter the number of years to convert:");
+
+    let buffer: String = get_input().expect("Invalid input. Provide only a positive integer.");
```

Comment:
```suggestion
    let buffer: String = get_input().expect("Could not read input.");
```
The message is misleading. This error happens when there is a problem getting the string from the standard input (it was closed while reading, or invalid utf-8).
https://doc.rust-lang.org/std/io/trait.BufRead.html#method.read_line